apps/site/assets/css/_event-overlay.scss

```diff
@@ -35,16 +35,29 @@
       background-color: transparent;
       border: 0;
       margin-left: 10px;
-      font-size: 18px;
+      font-size: 26px;
       height: 18px;
       line-height: 18px;
       font-weight: bold;
+      padding: 0;
     }
-  }
 
-  .m-event__title {
-    font-weight: bold;
-    margin-bottom: $base-spacing / 2;
+    .m-event__time-information {
+      display: flex;
+      flex: 1;
+      flex-direction: column;
+      justify-content: center;
```

Comment:
🤔 I am trying to vertically center the time info -- otherwise the time info looks off-center from the button.  The horizontal centering is correct as-is (even with the wide space on the right of the time info)